src/ontology/mondo-edit.obo

```diff
@@ -438255,6 +438256,7 @@ xref: MEDGEN:1636101 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: Orphanet:83482 {source="MONDO:equivalentTo"}
 xref: UMLS:C4707240 {source="MEDGEN:1636101", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0020067 {source="MONDO:Redundant", source="Orphanet:83482"} ! infectious encephalitis
+is_a: MONDO:1030003 {source="https://orcid.org/0000-0002-3458-4839"} ! mycoplasma infection
 intersection_of: MONDO:0020067 ! infectious encephalitis
 intersection_of: disease_has_infectious_agent NCBITaxon:2104 ! Mycoplasmoides pneumoniae
```

Comment:
@twhetzel I think this infectious agent is not correct, and should be (maybe?) changed to "Mycoplasma pneumoniae". (please check)
The reason it caught my attention is because "Mycoplasmoides pneumoniae" is not a "Mycoplasma", and therefore I questioned whether it should be a child. But I think the annotation is what is incorrect.

Reply:
Mycoplasma pneumoniae is a basionym (the original name on which a new name is based) for Mycoplasmoides pneumoniae (https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=info&id=2104) so this looks correct to me.

Reply:
@twhetzel: the reason why I am questioning this is because Mycoplanoides pneumoniae is not classified as a Mycoplasma. 

Therefore, if "Mycoplasma infection" is defined as having infectious agent Mycoplasma, 'Mycoplasma encephalitis' defined as having "infectious agent Mycoplanoides pneumoniae" should not be a child of "Mycoplasma infection" (since Mycoplanoides pneumoniae is not classified as a Mycoplasma, screenshot below). UNLESS we are saying that  'Mycoplasma encephalitis' has infectious agents both Mycoplanoides pneumoniae and Mycoplasma (but I don't think it is the case).

@nicolevasilevsky can you please take a look? Maybe I am missing something. 

![Screenshot 2024-09-09 at 7 46 09 PM](https://github.com/user-attachments/assets/ac1744c5-c8a0-45fc-8952-29cc4bb780a4)

![Screenshot 2024-09-09 at 7 46 29 PM](https://github.com/user-attachments/assets/190894fb-43c0-4983-a82a-0670df4e4904)

![Screenshot 2024-09-09 at 7 47 09 PM](https://github.com/user-attachments/assets/057fffcc-6b67-44e8-9182-de1f2912db48)

![Screenshot 2024-09-09 at 7 47 25 PM](https://github.com/user-attachments/assets/c4c95ef1-c9af-4e12-bf05-9fee851e3b1e)

Reply:
There was a re-naming over the last few years.
- https://pubmed.ncbi.nlm.nih.gov/29556819/
- https://pubmed.ncbi.nlm.nih.gov/31385780/
- https://pubmed.ncbi.nlm.nih.gov/31971499/

Since Mycoplasma pneumoniae is a basionym for Mycoplasmoides pneumoniae I'd gather these would still be called mycoplasma infections.

Reply:
I understand about the renaming. My issue is not about the renaming. My issue is that the taxon associated with the parent term is neither the same nor an ontological parent of the taxon associated with "mycoplasma encephalitis". 
If the taxon used for "mycoplasma encephalitis" is correct, then the taxon associated with the parent term should be reviewed. But as it currently is, it is not correct ontologically.

Reply:
I agree with Sabrina, I think the correct NCBIT term is Mycoplanoides pneumoniae, according to my [google search](https://www.google.com/search?q=primary+infectious+agent+for+mycoplasm+infection&rlz=1C1GCEU_enUS1113US1114&oq=primary+infectious+agent+for+mycoplasm+infection&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIJCAEQIRgKGKABMgkIAhAhGAoYoAEyCQgDECEYChirAtIBCDc2ODJqMGo3qAIAsAIA&sourceid=chrome&ie=UTF-8), this seems to be the primary infectious agenda in mycoplasma infections:

![image](https://github.com/user-attachments/assets/403fb1fb-3649-43bc-a9f7-772c82973232)


See:
https://www.ncbi.nlm.nih.gov/books/NBK536927/

Reply:
With the change in the naming, there is no longer a Mycoplasma pneumoniae, but a Mycoplasmoides pneumoniae (https://www.ebi.ac.uk/ols4/ontologies/ncbitaxon/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCBITaxon_2104?lang=en). 

I have another suggestion I will post shortly.

Reply:
To account for the change in names in NCBI Taxonomy from Mycoplasma pneumoniae to Mycoplasmoides pneumoniae I changed the new term name from Mycoplasma infection to Mycoplasmoides infection with a related synonym of mycoplasma infection. 

The individual disease names however do still include the word mycoplasma. For example as found in [DOID](https://www.ebi.ac.uk/ols4/ontologies/doid/classes?obo_id=DOID%3A13276) and [Orphanet](https://www.orpha.net/en/disease/detail/83482).